internal/pkg/service/buffer/store/key/id.go

```diff
@@ -0,0 +1,82 @@
+package key
+
+import (
+	"fmt"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+type (
+	ProjectID  int
+	ReceiverID string
+	ExportID   string
+	RevisionID int
+	FileID     UTCTime
+	SliceID    UTCTime
+)
+
+func (v ProjectID) String() string {
+	if v == 0 {
+		panic(errors.New("projectID cannot be empty"))
+	}
+	return fmt.Sprintf("%08d", v)
+}
+
+func (v ReceiverID) String() string {
+	if v == "" {
+		panic(errors.New("receiverID cannot be empty"))
+	}
+	return string(v)
```

Comment:
V `schema` pkg sme mali tieto podmienky, ze nastane `panic` ak sa zabudne vyplnit nejaky field.

No okrem toho mame (tu) metody `<key>.String()` kde to nebolo, a dost dlho som debugoval jeden problem, kym som zistil, ze tam nie je nastavena hodnota.

Preto som podmienky presunul sem a platia teda v celej apke aj v `schema`.